sample/benchmark/build.gradle.kts

```diff
@@ -31,12 +32,31 @@ android {
         java17()
     }
 
+    val bmPassword = System.getenv("BM_STORE_PASSWD")
+    signingConfigs {
+        if (bmPassword != null) {
+            create("release") {
+                storeFile = File(project.rootDir, "sample-benchmark.keystore")
+                storePassword = bmPassword
+                keyAlias = "dd-sdk-android"
+                keyPassword = bmPassword
+            }
+        }
+    }
+
     buildTypes {
-        release {
+        getByName("debug") {
             isMinifyEnabled = false
+        }
+
+        getByName("release") {
             proguardFiles(
                 getDefaultProguardFile("proguard-android-optimize.txt")
             )
+            isMinifyEnabled = true
+            if (bmPassword != null) {
```

Comment:
instead of checking the password maybe you should check if that `release` config exists ?

Reply:
replaced by code below:
```
signingConfigs.findByName("release")?.let {
          signingConfig = it
 }

```